tests/tests_assemble_workflow/test_bundle_recorder.py

```diff
@@ -118,7 +118,7 @@ def test_record_component_public(self):
                 "components": [
                     {
                         "commit_id": "3913d7097934cbfe1fdcf919347f22a597d00b76",
-                        "location": "https://ci.opensearch.org/ci/os-distro-prod/builds/1.1.0/c3ff7a232d25403fa8cc14c97799c323/x64/plugins",
+                        "location": "https://ci.opensearch.org/ci/os-distro-prod/artifacts/plugins",
```

Comment:
The base url should be changed to something like `https://ci.opensearch.org/ci/ci-env-prod/job-name-opensearch-distro/build-123/arch-amd64/platform-mac/` to be more inline with how this will be used in production